src/test/java/org/jabref/logic/exporter/BibtexDatabaseWriterTest.java

```diff
@@ -289,7 +289,56 @@ void writeEntryWithCustomizedTypeAlsoWritesTypeDeclaration() throws Exception {
     }
 
     @Test
-    void roundtrip() throws Exception {
+    void writeEntryWithConstantMonthApril() throws Exception {
+        BibEntry entry = new BibEntry.Builder(StandardEntryType.Misc)
```

Comment:
These tests should go to the `BibEntryWriterTest` as they are more concerned with a single entry than a complete database